docs/hud_data_model.md

```diff
@@ -0,0 +1,118 @@
+# HMIS HUD CSV data model
+
+Based on:
+HMIS CSV FORMAT Specifications FY2024
+VERSION 1.4
+
+## Key Relationships
+
+* One Organization can have many Projects
+* One Project can have many Enrollments
+* One Client can have many Enrollments
+* Each Enrollment is tied to exactly one Project and one Client
+* Each Enrollment can have multiple associated services, assessments, and other records
+
+## Enrollment ERD
+
+Note "CoC" and "Household" are not formally defined but are implied or virtual entities
+```mermaid
+erDiagram
+  Project }o--o{ CoC : "via ProjectCCoC"
+
+  CustomService }|--|| Enrollment : "has"
+  CustomServiceType ||--|{ CustomService : "has"
+  CustomServiceCategory ||--|{ CustomService : "has"
+
+
+
+  CustomAssessment }|--|| Enrollment : "has"
+  CustomCaseNote }|--|| Enrollment : "has"
+  CustomClientName }|--|| Client : "has"
+  CustomClientAddress }|--|| Client : "has"
+  CustomClientContactPoint }|--|| Client : "has"
+
+
+  CoC }|..|| Enrollment : "served in"
+  Project ||--|| Enrollment : "enrolled in"
+
+
+  Enrollment }|..|| Household: "in"
+
+
+  Enrollment ||--o| Exit : "may have"
```

Comment:
Arguably many of these are "may have".  I think I understand the reasoning for calling out Exit differently, but I wonder if we should indicate the optional nature across the board (based on HUD spec) or if we should just change this to "has".

Project has similar optional relationships.

Reply:
fair, I changed it to "has"

Perhaps these labels should be more descriptive? Something like:
* Exit "terminates" enrollment
* Enrollment "reports" or "records" Assessments, CLS, Income, etc

or we could take out the labels entirely if they aren't useful (`Enrollment ||--o| Exit : ""`)